core/modules/node/views/views_plugin_row_node_view.inc

```diff
@@ -50,10 +51,15 @@ class views_plugin_row_node_view extends views_plugin_row {
       '#options' => $options,
       '#title' => t('Display mode'),
       '#default_value' => $this->options['view_mode'],
-     );
+    );
+    $form['title'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Display title'),
+      '#default_value' => $this->options['title'],
+    );
     $form['links'] = array(
       '#type' => 'checkbox',
-      '#title' => t('Display links'),
+      '#title' => t("Display links (e.g. 'Read more')"),
```

Comment:
```suggestion
      '#title' => t('Display links (e.g. "Read more")'),
```

Reply:
I don't see any 'Reply' links, but there's an 'Add comment' one, so I've used that terminology instead.